cronsul

```diff
@@ -39,9 +52,8 @@ esac
 
 command=("$@")
 
-# This will be wrong if cron starts our job really late.
-schedule_date="$(( ($(date '+%s') / period) * period ))"
+current_run="$(date '+%s')"
```

Comment:
We could do that! I sort of wanted to change it because I didn't like how locks would last only a very short period of time as you got towards the end of the period. So I guess I don't see abandoning the old behavior has a bug. :wink: But if you'd rather keep it the same I can restore it.